command/server.go

```diff
@@ -530,6 +549,46 @@ func (c *ServerCommand) setupTelementry(config *server.Config) error {
 	return nil
 }
 
+func (c *ServerCommand) Reload(configPath []string) error {
```

Comment:
Do we need to expose this method?

Also, is it possible to use this method to load the configuration the first time as well?